services/webnn/ort/error_ort.h

```diff
@@ -24,6 +24,31 @@ namespace webnn::ort {
     }                                                                \
   } while (0);
 
+#define GET_STATUS_PTR(ort_func)                                   \
```

Comment:
`LOG_IF_FAILED(ort_func)`?

Reply:
I prefer `ORT_CALL_FAILED`, it indicates the returned value is a `boolean`, wdyt?

Reply:
Sorry for confusion. I meant keep this macro as its main purpose is to log if failed. And introduce another macro (ORT_CALL_FAILED) that returns a boolean, it can be used in cases like
```
if (ORT_CALL_FAILED(ort_call)) { return Error; }
```

Reply:
Updated in latest commit, PTAL~ Thanks!